cabal-install/src/Distribution/Client/CmdOutdated.hs

```diff
@@ -301,8 +301,14 @@ depsFromNewFreezeFile verbosity mprojectFile = do
   let ucnstrs = map fst . projectConfigConstraints . projectConfigShared
                 $ projectConfig
       deps    = userConstraintsToDependencies ucnstrs
+      freezeFile = distProjectFile distDirLayout "freeze"
+  freezeFileExists <- doesFileExist freezeFile
+  
+  unless freezeFileExists $
+    die' verbosity $
+      "Freeze file " ++ freezeFile ++ " does not exist"
```

Comment:
What about a little bit more detailed message (why the command needs the freeze file, how we can generate it)?

Reply:
Good point, will add that!

Reply:
@jneira pls review :)